platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -378,44 +380,35 @@ public boolean update(String pid, Map<String, Object> configurationTable) throws
   public void updateForLocation(
       final String pid, String location, Map<String, Object> configurationTable)
       throws IOException {
-    if (pid == null || pid.length() < 1) {
-      throw loggedException(ILLEGAL_PID_MESSAGE);
-    }
-    if (configurationTable == null) {
-      throw loggedException("Argument configurationTable cannot be null");
-    }
-
-    Configuration config = configurationAdmin.getConfiguration(pid, location);
+    if (pid == null || pid.length() < 1) throw loggedException(ILLEGAL_PID_MESSAGE);
```

Comment:
✏️ Should be properly indented and using `{}` to surround the block even if there is one statement. Not doing so can be error prone.

Reply:
Aww but it looked so nice without the extra noise 😛

Reply:
https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces